projects/storefrontlib/src/cms-components/navigation/footer-navigation/scroll-to-top/scroll-to-top.component.scss

```diff
@@ -0,0 +1,34 @@
+.cx-main {
+  position: fixed;
+  bottom: 15px;
+  right: 15px;
+  opacity: 0;
```

Comment:
When using opacity, please look closely on a fact, that when you are scrolled all the way up, you can still click the button. Change it to `display: none/block`. Also remove `transition` property, since it won't work with `display`.